Documentation/doc/Documentation/Installation.txt

```diff
@@ -1,1198 +0,0 @@
-/*!
-\page installation Installation
```

Comment:
I think the `installation` page should not be removed, to avoid that https://doc.cgal.org/latest/Manual/installation.html points to a 404 error. It should be instead a page explaining that CGAL is now header-only only, and point to the "usage" manual.